expr/functions/removeBetweenPercentile/function.go

```diff
@@ -0,0 +1,107 @@
+package removeBetweenPercentile
+
+import (
+	"context"
+	"fmt"
+	"math"
+
+	"github.com/go-graphite/carbonapi/expr/consolidations"
+	"github.com/go-graphite/carbonapi/expr/helper"
+	"github.com/go-graphite/carbonapi/expr/interfaces"
+	"github.com/go-graphite/carbonapi/expr/types"
+	"github.com/go-graphite/carbonapi/pkg/parser"
+)
+
+type removeBetweenPercentile struct {
+	interfaces.FunctionBase
+}
+
+func GetOrder() interfaces.Order {
+	return interfaces.Any
+}
+
+func New(configFile string) []interfaces.FunctionMetadata {
+	res := make([]interfaces.FunctionMetadata, 0)
+	f := &removeBetweenPercentile{}
+	functions := []string{"removeBetweenPercentile"}
+	for _, n := range functions {
+		res = append(res, interfaces.FunctionMetadata{Name: n, F: f})
+	}
+	return res
+}
+
+// removeBetweenPercentile(seriesLists, percent)
+func (f *removeBetweenPercentile) Do(ctx context.Context, e parser.Expr, from, until int64, values map[parser.MetricRequest][]*types.MetricData) ([]*types.MetricData, error) {
+	args, err := helper.GetSeriesArg(ctx, e.Args()[0], from, until, values)
+	if err != nil {
+		return nil, err
+	}
+
+	number, err := e.GetFloatArg(1)
+	if err != nil {
+		return nil, err
+	}
+
+	var percentile float64
+	var results []*types.MetricData
+
+	if number < 50.0 {
+		percentile = 100.0 - number
+	} else {
+		percentile = number
+	}
+
+	var lowerThresholds []float64
+	var higherThresholds []float64
+
+	for i := range args[0].Values {
+		var values []float64
+		for _, arg := range args {
+			if !math.IsNaN(arg.Values[i]) {
+				values = append(values, arg.Values[i])
+			}
+		}
+		if len(values) > 0 {
+			lowerThresholds = append(lowerThresholds, consolidations.Percentile(values, (100.0-percentile), false))
+			higherThresholds = append(higherThresholds, consolidations.Percentile(values, percentile, false))
+		}
+	}
+
+	for i, a := range args {
+		r := a.CopyLink()
+		r.Name = fmt.Sprintf("%s(%s, %g)", e.Target(), a.Name, number)
```

Comment:
May be without Sprintf, number can be converted to string outside a cycle. But not required, may be need benchmark for see costs of them.